fenics_ice/solver.py

```diff
@@ -1063,6 +1070,29 @@ def float_conditional(self, H, H_float=None):
 
         return fl_ex
 
+    def bglen_data_conditional(self, H, glen_mask, H_float=None):
+        """Compute a ufl Conditional where floating=1, grounded=0"""
+
+        if not self.params.ice_dynamics.allow_flotation:
+            fl_beta_mask = ufl.operators.Conditional(self.model.bglen_mask == 1, 
+                                          Constant(1.0, cell=triangle, name="Const data"),
+                                          Constant(0.0, cell=triangle, name="Const no data"))
```

Comment:
`> 0.0` in the conditional, for consistency with below.

Reply:
have modified the 2nd conditional to `bglen_mask == 1` instead